packages/ia-topnav/src/media-slider.js

```diff
@@ -0,0 +1,49 @@
+import { LitElement, html } from 'lit-element';
+import './media-subnav';
+import mediaSliderCSS from './styles/media-slider';
+
+class MediaSlider extends LitElement {
+  static get styles() {
+    return mediaSliderCSS;
+  }
+
+  static get properties() {
+    return {
+      config: { type: Object },
+      mediaSliderOpen: { type: Boolean },
+      selectedMenuOption: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.config = {};
+    this.mediaSliderOpen = false;
+    this.selectedMenuOption = 'texts';
+  }
+
+  shouldUpdate() {
+    const scrollPane = this.shadowRoot.querySelector('.information-menu');
+
+    if (!scrollPane) { return true; }
+    scrollPane.scrollTop = 0;
+    return true;
```

Comment:
Since you're returning true regardless, you could just set `scrollTop` when it exists:
```suggestion
    if (scrollPane) {
      scrollPane.scrollTop = 0;
    }
    return true;
```

Reply:
Yeah, this is a `return true` that was required by ESLint that I just added in to shut it up. Previously, I only had one return statement.